buildspec.yml

```diff
@@ -0,0 +1,35 @@
+version: 0.1
+
+environment_variables:
+  plaintext:
+    DATE_FORMAT: "%F %H:%M:%S"
+
+phases:
+  install:
+    commands:
+      - echo "$(date +"${DATE_FORMAT}") - Begin install phase (pre-build)"
+      - echo "Install jscs..."
+      - npm install -g jscs
+  pre_build:
+    commands:
+      - echo "$(date +"${DATE_FORMAT}") - Start build"
+      - echo "AWS CodeBuild src directory: ${CODEBUILD_SRC_DIR}"
```

Comment:
Yet another YAML quirk. Thanks, @paulkerschen  I'm going to wrap all commands in quotes and remove the extraneous.